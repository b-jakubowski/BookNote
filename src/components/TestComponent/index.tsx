import { StyleSheet, Text, View } from 'react-native';

export default function TestComponent() {
	return (
		<View style={styles.container}>
			<Text>TestComponent</Text>
		</View>
	);
}

const styles = StyleSheet.create({
	container: {
		width: 200,
		height: 200,
		backgroundColor: 'red',
	},
});
