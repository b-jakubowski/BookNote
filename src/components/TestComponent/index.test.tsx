import React from 'react';
import { render } from '@testing-library/react-native';
import TestComponent from '.';

describe('Button', () => {
	it('should match snapshot', () => {
		expect(render(<TestComponent />).toJSON()).toMatchSnapshot();
	});
});
