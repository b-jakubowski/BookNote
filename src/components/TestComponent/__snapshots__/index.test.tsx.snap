// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should match snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "red",
      "height": 200,
      "width": 200,
    }
  }
>
  <Text>
    TestComponent
  </Text>
</View>
`;
